docs/zkapps/tutorials/10-account-updates.mdx

```diff
@@ -0,0 +1,381 @@
+
+### Running our Smart Contracts and Visualizing the AccountUpdates
+
+We'll now discuss the file [main.ts](https://github.com/o1-labs/docs2/tree/main/examples/zkapps/10-account-updates/src/main.ts), which creates transactions with the above smart contracts, and the account update visualizations it creates.
+
+First, we'll import the transaction visualizer:
+
+```typescript
+...
+import { showTxn, saveTxn, printTxn } from 'mina-transaction-visualizer';
+...
+```
+
+This provides us 3 functions:
+
+```typescript
+// creates a png file of a transaction, and opens it in a local image viewer
+async showTxn(txn: Mina.Transaction, name: string, legend: Legend)
+
+// creates a png file of a transaction, and saves it to a path
+saveTxn(txn: Mina.Transaction, name: string, legend: Legend, path: string)
+
+// prints a nicely formatted view of a transaction
+showTxn(txn: Mina.Transaction, name: string, legend: Legend)
+
+// with legend type, to replace public keys with human readable strings:
+type Legend = { [pk: string]: string };
+
+```
+
+We'll build our legend as follows:
+
+```typescript
+  const legend = {
+    [proofsOnlyAddr.toBase58()]: 'proofsOnlyZkApp',
+    [secondaryAddr.toBase58()]: 'secondaryZkApp',
+    [deployerAccount.toPublicKey().toBase58()]: 'deployer',
+  };
+```
+
+Then we'll deploy our smart contracts as follows, and visualize the transaction:
+
+```typescript
+  const deploy_txn = await Mina.transaction(deployerAccount, () => {
+    AccountUpdate.fundNewAccount(deployerAccount, 2);
+    proofsOnlyInstance.deploy({ zkappKey: proofsOnlySk });
+    secondaryInstance.deploy({ zkappKey: secondarySk });
+  });
+
+  await deploy_txn.prove();
+  deploy_txn.sign([ proofsOnlySk, secondarySk ]);
+
+  await showTxn(deploy_txn, 'deploy_txn', legend);
+
+  await deploy_txn.send();
+```
+
+This yields the following visualization of `deploy_txn`. 
+
+![](https://raw.githubusercontent.com/es92/zkApp-examples/main/10-account-updates/deploy_txn.png)
+
+This is best opened in a new tab to read the contents ([link](https://raw.githubusercontent.com/es92/zkApp-examples/main/10-account-updates/deploy_txn.png)).
+
+The deploy transaction includes 5 accountUpdates. In order from left to right;
+
+1. This accountUpdate takes the new account fee from the deployer for deploying the zkApps. Note the `-2` on the `balanceChange` field
+2. This update deploys the `proofsOnlyZkApp` instance. Note the permissions, all set to the values in the zkApp's deploy field, and the `preconditions`, asserting its a new account.
+3. This update initializes the `proofsOnlyZkApp`. Note the precondition that it can't already be in a proved state.
+4. This update deploys an instance of `secondaryZkApp`. Note the permissions here set to default compared to the deployment in the `proofsOnlyZkApp` example.
+5. This update initializes the `secondaryZkApp` instance.
```

Comment:
there's a nuanced thing going on here: because you annotated `init()` with `@method`, there is a separate account update after the deploy one which performs the `init()` logic and has a proof. If you would've left out `@method`, the init stuff would've been part of the deploy update.

Could be interesting to mention that or even make the second method do `init()` without `@method` to highlight the difference. If you think that's worth explaining